packages/twenty-front/src/modules/workflow/search-variables/utils/getVariableTag.ts

```diff
@@ -0,0 +1,50 @@
+import { Node } from '@tiptap/core';
+import { mergeAttributes, RawCommands } from '@tiptap/react';
+
+export const getVariableTag = () =>
+  Node.create({
+    name: 'variableTag',
+    group: 'inline',
+    inline: true,
+    atom: true,
+
+    addAttributes: () => ({
+      variable: {
+        default: null,
+      },
+    }),
+
+    renderHTML: ({ node, HTMLAttributes }) => {
+      const variable = node.attrs.variable as string;
+      const variableWithoutBrackets = variable.replace(
+        /\{\{([^}]+)\}\}/g,
+        (_, variable) => {
+          return variable;
+        },
+      );
+
+      const parts = variableWithoutBrackets.split('.');
+      const displayText = parts[parts.length - 1];
```

Comment:
I'm wondering whether we couldn't display more explicit text than the last part of the variable path. Indeed, we might have several emails at different paths. It might be more explicit to print the whole path, i.e., `input.properties.after.email`. I'm not sure if it's the most aesthetic choice, but I think it can be the least confusing at the end.

https://github.com/user-attachments/assets/e9d780da-5bce-4681-a95f-496499b1a554

Reply:
will discuss that with @Bonapara!

Reply:
Think it would make sense, we could enlarge the input and display the path in the menu header.